src/operator.rs

```diff
@@ -588,7 +583,7 @@ impl Parse {
             })
         } else {
             Result::Ok(Parse {
-                regex: regex::Regex::new(&regex_str).unwrap(),
+                regex: lang::Keyword::new_with_wildcard(pattern.to_string()),
```

Comment:
it might be a bit cleaner to just make the code in a Keyword helper code for regexes instead of keeping it within the Keyword class.

Either that or just say `.to_regex` here rather than later -- I don't really see a compelling reason to store a Keyword object in the parser instead of just a raw regex.

Reply:
Indeed, I wasn't paying close enough attention to how things were organized and how you had kept the lang module separated from this module.  I've rejiggered things some more to try to follow the existing structure.  The Regex is no longer built during parsing, that is now done when converting the parse tree to filters/operators.  Along the way, I ended up allowing for more than one filter so that I could remove the Search enum and make some of the guts a bit simpler.